### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mchv**
## What does this change?

The associated changes improve our engineering culture documentation expanding few items we value _significantly_:
- `Delivery mindset`
- `Data informed`
- `Empowerment`
- `Knowledge sharing`

It, as well,  try to be more explicit for each item on the expected behaviours associated to each value. Before writing those changes, we have reviewed each team value and behaviours, and ensure those we end up are the one we _currently_ value and have valued for a _long time_.

One of the goals behind adding behaviours and being more explicit is to help new starters integrate the department and feeling  comfortable. Feedback from @mxdvl @i-hardy and @rhystmills will be helpful.     

I think the associated changes are a great addition, and I particularly appreciate that we explicitly mention pairing, which is a practice we encourage, and some people like @rtyley and @amyhughes have been great ambassadors.

@guardian/guardian-developers-write Before merging this, I would like to ensure, there is nothing people have a specific issue with. As this an open repository, if there is something you would prefer to privately feedback feel free to email me.     
   
Because culture is something we share as well with previous employees who have helped created it, I think it is worth tagging @sallygoble @rrees @tackley @jennysivapalan @philwills @LATaylor-guardian @gklopper @NataliaLKB

### CODE REVIEW

**Comment by gtrufitt** [comment]
Are we Data *Driven* or Data *Informed*?

File: README.md [hunk 1]
```diff
@@ -1,33 +1,80 @@
...
->We choose to work here because we believe in what the Guardian stands for. We're excited at contributing to the Guardian's future. 
+_Behaviours_:
+ - We fail and succeed together as a team.
+ - We are independent, accountable and share responsibility. We conduct ourselves with humility, kindness and a calm manner. We debate ideas, not people and we share constructive feedback openly. 
+ - We take the time needed to listen to and help our colleagues with any problem they might have from anywhere within the business.
+ - We value the experience of people in the team and the different constraints within their lives. 
+ - We celebrate success and positive behaviour, recognizing everyone’s contribution while treating every moment including tasks and failures as a learning opportunity.
 
-We believe in treating everyone we encounter with respect.
+**We are data driven**
```
Comment (gtrufitt) [inline README.md]: Are we Data *Driven* or Data *Informed*?

One definition:

https://segment.com/resources/data-strategy/data-driven-vs-data-informed/

> Data-driven means that you always rely on data to make your decisions. If you’re trying to improve the UX of your product, you’ll run tests to get data to find the best approach.

> Data-informed means that you use data alongside your unique experience, user research, and other inputs to make decisions. With a data-informed approach, data is just one part of your decision-making process.

I believe that we're much more aligned to the Data Informed approach - we use the data to help make decisions alongside the historic, editorial, user experience and performance contexts.

File: README.md [hunk 2]
```diff
@@ -1,33 +1,80 @@
...
-> We can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting emerging tools and solutions before others. We maintain our technical advantage through continuous innovation.
+**Technical Excellence** 
 
-We care.
+We believe that embracing technical innovation and excellence can help us make a difference.
 
->We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
+_Behaviours_:
+- We think we can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting some emerging tools and solutions before others:
+    - We have been one of the [first company in UK to adopt `scala`](https://www.theguardian.com/info/developer-blog/2013/apr/26/how-the-guardian-built-scalable-foundation)
```
Comment (buck06191) [inline README.md]:
```suggestion
    - We were one of the [first companies in UK to adopt `scala`](https://www.theguardian.com/info/developer-blog/2013/apr/26/how-the-guardian-built-scalable-foundation)
```
Grammar fix

File: README.md [hunk 3]
```diff
@@ -1,33 +1,80 @@
...
+    - We have been one of the first company [to use continuous delivery](https://www.theguardian.com/info/developer-blog/2015/jan/05/delivering-continuous-delivery-continuously)
```
Comment (buck06191) [inline README.md]:
```suggestion
    - We were one of the first companies [to use continuous delivery](https://www.theguardian.com/info/developer-blog/2015/jan/05/delivering-continuous-delivery-continuously)
```

File: README.md [hunk 4]
```diff
@@ -1,33 +1,80 @@
...
+    - We have been one of the [first legacy company to move entirely to the cloud (AWS)](http://perspectives.mvdirona.com/2016/03/a-decade-of-innovation/)
```
Comment (buck06191) [inline README.md]:
```suggestion
    - We were one of the [first legacy companies to move entirely to the cloud (AWS)](http://perspectives.mvdirona.com/2016/03/a-decade-of-innovation/)
```

File: README.md [hunk 5]
```diff
@@ -1,33 +1,80 @@
...
+    - We have been one of the pioneer of [the concept of microservices](https://martinfowler.com/articles/microservices.html#footnote-etymology)
```
Comment (buck06191) [inline README.md]:
```suggestion
    - We were a pioneer of [the concept of microservices](https://martinfowler.com/articles/microservices.html#footnote-etymology)
```

File: README.md [hunk 6]
```diff
@@ -1,33 +1,80 @@
...
+ - We maintain our technical advantage through continuous innovation
+ - We aim to be at the forefront of tech, staying curious, looking to innovate and continuously improve.
 
+**Knowledge sharing**
 
+We believe that it’s our responsibility to upskill one another, sharing our knowledge, learnings and mistakes. 
 
+_Behaviours_:
+ - Individuals are entitled to pairing time for their development.
```
Comment (buck06191) [inline README.md]: It would be nice to emphasise that we also are dedicated to developing best practice for pairing. Good quality pairing that isn't passive or unequal in "burden" is really important, especially at a junior level

File: README.md [hunk 7]
```diff
@@ -1,33 +1,80 @@
...
 
+_Behaviours_:
+ - Individuals are entitled to pairing time for their development. 
+ - We make time for individual-learning & career development with personal development being our priority.
+ - We write code for the most junior, not the most senior.
+ - We actively share at all levels of users and stakeholders, whether it's our knowledge, work, planning or approach.
 
+**We care**
+ 
+We choose to work here because we believe in [its values](https://workforus.theguardian.com/our-values) and what the Guardian stands for. We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
```
Comment (buck06191) [inline README.md]:
```suggestion
We choose to work here because we believe in [our values](https://workforus.theguardian.com/our-values) and what the Guardian stands for. We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
```

Makes the values more inclusive and matches the link location

**Review by buck06191 — COMMENTED** [review]
A few grammar fixes and a suggestion for elaborating on the pairing section

File: README.md (hunk 6 above)
Comment (jennygrahamjones) [reply README.md]: I'm not sure how best to word this, but could we say that we encourage pairing, rather than that we are entitled to it?

File: README.md [hunk 8]
```diff
@@ -1,33 +1,80 @@
...
+
+**Inclusiveness & Respect** 
 
 We believe that our culture is important, and we all contribute to it regardless of our role or level.
 
->Every one of us has multiple interactions with a variety of people across the department and company every day. Each one of those is an opportunity to express and refine our culture. This written statement of our culture is shared, and anyone can [contribute](CONTRIBUTING.md).
+_Behaviours_:
+ - We ensure to treat everyone we encounter with respect independently of their skillsets and levels of ability
+ - Every one of us has multiple interactions with a variety of people across the department and company every day. Each one of those is an opportunity to express and refine our culture. This written statement of our culture is shared, and anyone can [contribute](CONTRIBUTING.md).
+ - We respect the opinion of others and are gracious in our interactions, whether meeting physically or digitally, whether friend or stranger, whether employee, customer or commenter.
```
Comment (johnduffell) [inline README.md:18]: I would personally restrict respect to opinions about product and engineering and other generic topics etc,.  It doesn't come up often but presumably we want to be explicit about not protecting uninclusive opinions expressed.

File: README.md [hunk 9]
```diff
@@ -1,33 +1,80 @@
...
+ - We come from different backgrounds, work in different roles, are different ages, and therefore have different levels of experience. All in all, we pull each other up, not bring each other down.
```
Comment (johnduffell) [inline README.md]: someone reading this might feel it's a bit "etc" on the underrepresented group(s) they identify with, if we care we can specify more groups we would welcome in this point

File: README.md (hunk 5 above)
Comment (johnduffell) [reply README.md]: was that footnote anchor the intended destination of the link?

**Comment by johnduffell** [comment]
yes I think this is great and definitely due a refresh.  I don't think everything in there is valued the same in all teams but that's no bad thing.  I think we could be a bit more heavy on the diversity thing, as that is an important part that we focus on a lot.

File: README.md (hunk 9 above)
Comment (mchv) [reply README.md]: I think the mention of `background`, `roles` and `ages` is linked here to support the different `levels of experience`. I am not sure mentioned `underrepresented group(s)` will make this sentence particularly better, however I think we could another sentence.

File: README.md [hunk 10]
```diff
@@ -1,33 +1,81 @@
...
+ - We value and respect all differences in all people (seen and unseen). We aspire to inclusive working experiences and an environment that reflects the audience we serve, where our people have equal access to career development opportunities, their voices are heard and can contribute to our future. We aim to hire, retain and advance people from underrepresented groups.
```
Comment (mchv) [inline README.md]: @johnduffell what do you think of this suggestion, regarding your previous comment?

File: README.md (hunk 1 above)
Comment (mchv) [reply README.md]: @gtrufitt Thanks Gareth, great point! I have now amended to use `Data informed`

File: README.md (hunk 9 above)
Comment (amyhughes) [reply README.md]: How about we rephrase this to take out the group characteristics and instead say "Our experiences and backgrounds differ"  to address John's point above?

Also I think we can call out more than an appreciation of the impact this has on level of experience here, how about:

"and therefore so do our perspectives, areas of expertise, and levels of experience"

So all together:

"Our experiences and backgrounds differ, and therefore so do our perspectives, areas of expertise, and levels of experience. All in all ..."

File: README.md (hunk 6 above)
Comment (mchv) [reply README.md]: @buck06191 could you suggest a sentence to describe your thinking?
@jennygrahamjones Would `Individuals are encouraged to pair, as part of their development` be fine, or do you have a better suggestion?

Comment (mchv) [reply README.md]: @jennygrahamjones thanks, I have now updated.

File: README.md (hunk 9 above)
Comment (mchv) [reply README.md]: @amyhughes Thanks, I like your wording and have now used it!

File: README.md (hunk 6 above)
Comment (buck06191) [reply README.md]: @mchv Something along the lines of 
```
 - Individuals are encouraged to pair as part of their development and are empowered to take an active role in what this looks like and to work together to find their own best practice.
```

If someone has more concrete suggestions for best practice, that could be included, but I also think it's about empowering people to take charge of how they pair and what works best for them as I've found this varies depending on individuals, teams and the work being undertaken.

Comment (amyhughes) [reply README.md]: I think "encouraged" is a bit thin, "entitled" gave the sense that the department would ensure that pairing was available rather than it being up to the individual to try to forge this culture. The issue I have with "entitled" is that it makes pairing out to be something that is offered reluctantly.

I think we could do more to pull out the shared benefits of pairing here, currently it sound like we just see it as an aid to development. How about:

"We practice pairing to help share our knowledge, grow our skills, and facilitate collaborative decision making"

Comment (jennygrahamjones) [reply README.md]: @amyhughes that captures much more eloquently what I was thinking of with 'encouraged' :)

File: README.md (hunk 10 above)
Comment (johnduffell) [reply README.md]: great.  if we are happy to strengthen it to "does happen" rather than "can" that would be even better
```suggestion
 - We value and respect all differences in all people (seen and unseen). We aspire to inclusive working experiences and an environment that reflects the audience we serve, where our people take up career development opportunities equally, their voices are heard and contribute to our future. We aim to hire, retain and advance people from underrepresented groups.
```

File: README.md (hunk 6 above)
Comment (amyhughes) [reply README.md]: Sorry Josh, I've just noticed your amendment. How about a second sentence which says something like:

"We value pairing as a skill which we encourage people to grow, working together to find the styles which are most effective."

@rtyley might have something to add here

**Review by tjmw — COMMENTED** [review]
This looks great. Any thoughts on making this one of our pinned repos on GitHub to make it a bit more front and centre?

**Comment by rhystmills** [comment]
This feels clear and concise to me - and reflects my experience of the team so far.

File: README.md (hunk 6 above)
Comment (mchv) [reply README.md]: Thanks @amyhughes for the suggestions, I have now updated the wording.

File: README.md [hunk 11]
```diff
@@ -1,33 +1,82 @@
...
-We care.
+We believe that embracing technical innovation and excellence can help us make a difference.
 
->We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
+_Behaviours_:
+- We think we can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting some emerging tools and solutions before others:
+    - We were one of the [first companies in UK to adopt `scala`](https://www.theguardian.com/info/developer-blog/2013/apr/26/how-the-guardian-built-scalable-foundation)
+    - We were one of the first companies [to use continuous delivery](https://www.theguardian.com/info/developer-blog/2015/jan/05/delivering-continuous-delivery-continuously)
+    - We were one of the [first legacy companies to move entirely to the cloud (AWS)](http://perspectives.mvdirona.com/2016/03/a-decade-of-innovation/)
+    - We were a pioneer of [the concept of microservices](https://martinfowler.com/articles/microservices.html#footnote-etymology)
```
Comment (mxdvl) [inline README.md]: Slightly more consistent:
```suggestion
    - We were one of the [first companies in UK to adopt `scala`](https://www.theguardian.com/info/developer-blog/2013/apr/26/how-the-guardian-built-scalable-foundation)
    - We were one of the [first companies to use continuous delivery](https://www.theguardian.com/info/developer-blog/2015/jan/05/delivering-continuous-delivery-continuously)
    - We were one of the [first legacy companies to move entirely to the cloud (AWS)](http://perspectives.mvdirona.com/2016/03/a-decade-of-innovation/)
    - We were a pioneer of the [concept of microservices](https://martinfowler.com/articles/microservices.html#footnote-etymology)
```

**Review by mxdvl — COMMENTED** [review]
This was good to read through and I agree that making it a pinned repo as @tjmw suggested is a good idea. Would it be worth investigating in the [new special repo] for a better company presentation?

[new special repo]: https://dev.to/m0nica/how-to-create-a-github-profile-readme-1paj

**Comment by mchv** [comment]
@mxdvl the plan is as well to have it included and  in our [new engineering website](https://github.com/guardian/guardian-engineering-site) has a main section. There is still a lot of work before the new site is ready.

**Comment by tackley** [comment]
It has made my day to see such a brilliant, well though out set of updates to this ancient doc :) Wholeheartedly agree with all of the changes. The Guardian team culture remains the best I have ever had the privilege of working within - keep up the good work!

File: README.md [hunk 12]
```diff
@@ -1,33 +1,82 @@
...
+ - We are independent, accountable and share responsibility. We conduct ourselves with humility, kindness and a calm manner. We debate ideas, not people and we share constructive feedback openly. 
+ - We take the time needed to listen to and help our colleagues with any problem they might have from anywhere within the business.
+ - We value the experience of people in the team and the different constraints within their lives. 
+ - We celebrate success and positive behaviour, recognizing everyone’s contribution while treating every moment including tasks and failures as a learning opportunity.
 
-We believe in treating everyone we encounter with respect.
+**We are data informed** 
 
->Whether meeting physically or electronically, whether friend or stranger, whether employee, customer or commenter, we respect the opinion of others and are gracious in our interactions.
+We believe that to be responsible in our impact we must be led by data.
```
Comment (gtrufitt) [inline README.md]:
```suggestion
We believe that to be responsible in our impact we must be informed by data.
```

**Review by gtrufitt — APPROVED** [review]

**Comment by jranks123** [comment]
@mchv Would this be a good place to link to the AB testing document? Here is the link https://docs.google.com/presentation/d/1WhkEjeU8LWcFINRfQQxgZ7Bi8RqQOH_bmEawjWi65k8/edit?usp=sharing

File: README.md [hunk 13]
```diff
@@ -1,33 +1,82 @@
+**Delivery mindset**
+
+We believe in being pragmatic and moving fast, spending our energy on the right things, shipping code and then rapidly iterating on it. 
+
+_Behaviours_:
+- We simplify our work, focusing on getting it to the user as early as possible.
+- We practice [continuous delivery](https://en.wikipedia.org/wiki/Continuous_delivery)
+- We are trying to [make the change easy, before the easy change](https://twitter.com/KentBeck/status/250733358307500032).   
+- We aim to solve issues rapidly prioritising them based on impact they may have.
```
Comment (maxspencer) [inline README.md]: Suggest using "problems" instead of "issues" because I think it makes more sense to "solve a problem" than to "solve an issue". Also removed "may" from the second part.

```suggestion
- We aim to solve problems rapidly, prioritising them based on their impact.
```

File: README.md [hunk 14]
```diff
@@ -1,33 +1,82 @@
...
+**We are data informed** 
 
->Whether meeting physically or electronically, whether friend or stranger, whether employee, customer or commenter, we respect the opinion of others and are gracious in our interactions.
+We believe that to be responsible in our impact we must be led by data.
 
-We are respectful of different skill sets and levels of ability.
->We come from different backgrounds, work in different roles, are different ages, and therefore have different levels of experience. All in all, we pull each other up, not bring each other down.
+_Behaviours_: 
+ - We base our decisions on analytics and experiments to reach consensus having heard everyone’s voice. 
+ - We measure our success through data analysis and not through the highest paid person's opinion or the loudest voice.
```
Comment (maxspencer) [inline README.md:40]: I'd like to propose a new item here, it's something I think is important, let me know if you agree?

> When we can't make an informed decision we're ready to say "we don't know". Then it's time to gather whatever new information we need.

File: README.md [hunk 15]
```diff
@@ -1,33 +1,82 @@
...
+ - We make time for individual-learning & career development with personal development being our priority.
+ - We write code for the most junior, not the most senior.
+ - We actively share at all levels of users and stakeholders, whether it's our knowledge, work, planning or approach.
 
+**We care**
+ 
+We choose to work here because we believe in [our values](https://workforus.theguardian.com/our-values) and what the Guardian stands for. We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
 
+_Behaviours_: 
+ - We seek to be a pioneer in the way things are done in the business and and aim to be seen as a model by other teams and departments
```
Comment (maxspencer) [inline README.md]: Nitpick: full stop

```suggestion
 - We seek to be a pioneer in the way things are done in the business and and aim to be seen as a model by other teams and departments.
```

File: README.md [hunk 16]
```diff
@@ -1,33 +1,82 @@
...
+_Behaviours_:
+ - We practice pairing to help share our knowledge, grow our skills, and facilitate collaborative decision making.
+ - We value pairing as a skill which we encourage people to grow, working together to find the styles which are most effective.
+ - We make time for individual-learning & career development with personal development being our priority.
+ - We write code for the most junior, not the most senior.
+ - We actively share at all levels of users and stakeholders, whether it's our knowledge, work, planning or approach.
 
+**We care**
+ 
+We choose to work here because we believe in [our values](https://workforus.theguardian.com/our-values) and what the Guardian stands for. We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
```
Comment (maxspencer) [inline README.md]: To my mind, it would be better to shorten the first sentence:

```suggestion
We believe in [our values](https://workforus.theguardian.com/our-values) and what the Guardian stands for. We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. We are enthused to do whatever we can to deliver value to the Guardian.
```

Whether our values are _the_ reason we choose to work here is neither here nor there. If they are then that's fine, but I think there are a lot of other good reasons (many of them mentioned above) that one might choose to work here.

**Review by maxspencer — APPROVED** [review]
I've read through and added some comments which I hope are helpful.

**Review by katebee — APPROVED** [review]
👏  💛 👍

File: README.md [hunk 17]
```diff
@@ -1,33 +1,82 @@
...
+**We are data informed** 
 
->Whether meeting physically or electronically, whether friend or stranger, whether employee, customer or commenter, we respect the opinion of others and are gracious in our interactions.
+We believe that to be responsible in our impact we must be informed by data.
 
-We are respectful of different skill sets and levels of ability.
->We come from different backgrounds, work in different roles, are different ages, and therefore have different levels of experience. All in all, we pull each other up, not bring each other down.
+_Behaviours_: 
+ - We base our decisions on analytics and experiments to reach consensus having heard everyone’s voice. 
+ - We measure our success through data analysis and not through the highest paid person's opinion or the loudest voice.
```
Comment (rrees) [inline README.md:40]: This feels a bit disingenuous as it implies that data analysis is in some way objective. I suspect it also isn't true, the process of "data analysis" probably delivers something that allows a consensus to be generated about whether a change is successful.

File: README.md [hunk 18]
```diff
@@ -1,33 +1,82 @@
...
+**Empowerment**
 
->This behaviour is consistently encouraged across all levels of engineering.
+We believe that all of us want to - and can - make a difference to the Guardian.
 
-We believe that embracing technical innovation and excellence can help us make a difference
+_Behaviours_: 
+  - Meeting the needs of people we encounter across the Guardian is consistently encouraged across all levels of engineering.
+  - We trust engineers to build the right things, collaborating closely and transparently with other teams.
+  - We own the problem, not the spec. We don’t wait for decisions to be made for us, we are an empowered engineering function and you are an empowered engineer. We use that power responsibly as we are conscious some decisions may need to be signed off and work collaboratively with our stakeholders.
```
Comment (rrees) [inline README.md:49]: This should probably be broken down into smaller points. Some of them are also beliefs rather than behaviours

File: README.md [hunk 19]
```diff
@@ -1,33 +1,82 @@
...
-> We can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting emerging tools and solutions before others. We maintain our technical advantage through continuous innovation.
+**Technical Excellence** 
 
-We care.
+We believe that embracing technical innovation and excellence can help us make a difference.
 
->We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
+_Behaviours_:
+- We think we can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting some emerging tools and solutions before others:
+    - We were one of the [first companies in UK to adopt `scala`](https://www.theguardian.com/info/developer-blog/2013/apr/26/how-the-guardian-built-scalable-foundation)
```
Comment (rrees) [inline README.md:57]: This is your history not your behaviour

File: README.md [hunk 20]
```diff
@@ -1,33 +1,82 @@
...
+    - We were one of the [first companies to use continuous delivery](https://www.theguardian.com/info/developer-blog/2015/jan/05/delivering-continuous-delivery-continuously)
+    - We were one of the [first legacy companies to move entirely to the cloud (AWS)](http://perspectives.mvdirona.com/2016/03/a-decade-of-innovation/)
+    - We were a pioneer of the [concept of microservices](https://martinfowler.com/articles/microservices.html#footnote-etymology)
+ - We maintain our technical advantage through continuous innovation
+ - We aim to be at the forefront of tech, staying curious, looking to innovate and continuously improve.
```
Comment (rrees) [inline README.md]:
```suggestion
 - We will be at the forefront of tech, always being curious, innovating and continuously improving.
```
This seems like an objective, I've suggested more directive language

**Comment by rrees** [comment]
I think it's a great idea to refresh this but I'm not sure the structure is really working.

I think it might work better to put your aim or principles as the first paragraph. Then describe the behaviours you're looking for people to model. Then give examples of practices that reinforce that behaviour. I think this will make it easier to update and refine all these things in future.

## Example

### Principles

We value sharing knowledge across our teams

### Behaviours

We will make time to answer our colleagues questions
We will proactively shared ideas that we have learned

### Practices

Paring or mob programming
Techtime (is this still a thing?)

File: README.md [hunk 21]
```diff
@@ -1,33 +1,82 @@
...
+ - We write code for the most junior, not the most senior.
+ - We actively share at all levels of users and stakeholders, whether it's our knowledge, work, planning or approach.
 
+**We care**
+ 
+We choose to work here because we believe in [our values](https://workforus.theguardian.com/our-values) and what the Guardian stands for. We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
 
+_Behaviours_: 
+ - We seek to be a pioneer in the way things are done in the business and and aim to be seen as a model by other teams and departments.
+
```
Comment (mxdvl) [inline README.md:81]: Is that our only caring behaviour?

**Review by mxdvl — APPROVED** [review]

File: README.md (hunk 8 above)
Comment (mchv) [reply README.md:18]: As you know `uninclusive opinions` could be a divisive topic because outside of what is covered by law `uninclusive` is subject to interpretation based on culture, time and everyone's background. The Guardian is an independent newspaper and media organisation and as such may choose to publish articles or not moderate comments that may be felt by some as `uninclusive`. As an employee one have the right to (strongly) disagree with some editorial choice, a moderation decision, a customer review, or with an other employee's opinion but that disagreement should be expressed in a way which is respectful to the other party.

File: README.md (hunk 17 above)
Comment (mchv) [reply README.md:40]: This is a fair point as objectivity is difficult(impossible) to achieve and one could always game a system to achieve what they would like. However I think this sentence is explicit enough on how we aim to measure success without defining a flawless protocol to do so.

File: README.md (hunk 18 above)
Comment (mchv) [reply README.md:49]: Any suggestion on how you would rewrite it?

File: README.md (hunk 19 above)
Comment (mchv) [reply README.md:57]: I have rewrote it to made clearer this history illustrate the behaviour

**Comment by mchv** [comment]
@rrees Thanks for the suggestion on the structure. We started with a difference between `behaviours` and `practices` but we found that differentiation not always working and helping whole readability of the document. We may however update that document in the future if we found more to add and that distinction more helpful at that time.

File: README.md (hunk 14 above)
Comment (mchv) [reply README.md:40]: @maxspencer While I fully agree with the first part of the sentence, I think the second part is more nuanced. we sometime make decision even if we are uninformed or not enough because we have to move on. I did not came with a great wording so feel free to suggest one in a new PR

File: README.md (hunk 21 above)
Comment (mchv) [reply README.md:81]: Feel free to suggest other ones with a new PR
